File: modul3/C/PRAK302-2310817210023-RifkyPutraMahardika.c
#include <stdio.h>
int main() {
    int nilai;
    printf("Nilai : ");
    scanf("%d", &nilai);

    if (nilai >= 80){
        printf("Huruf : A");}
    else if (nilai < 80, nilai > 69 ){
        printf("Huruf : B");}
    else if (nilai < 70, nilai > 59){
        printf("Huruf : C");}
    else if (nilai < 60, nilai > 49){
        printf("Huruf : D");}
    else if (nilai < 50){
        printf("Huruf : E");}

return 0;
}

File: modul3/Python/PRAK302-2310817210023-RifkyPutraMahardika.py
nilai = int(input("Nilai : "))
if nilai >= 80:
    print("Huruf : A")
elif nilai > 69 and nilai < 80:
    print("Huruf : B")
elif nilai > 59 and nilai < 70:
    print("Huruf : C")
elif nilai > 49 and nilai < 60:
    print("Huruf : D")
elif nilai < 50:
    print("Huruf : E")